pages/Smeltery.md

```diff
@@ -0,0 +1,11 @@
+# Smeltery
```

Comment:
Wiki pages should not include headers.
They have headers automatically, this would create two.